README.md

```diff
@@ -6,47 +6,60 @@
 ![Cargo format](https://github.com/google/OpenSK/workflows/Cargo%20format/badge.svg?branch=develop)
 [![Coverage Status](https://coveralls.io/repos/github/google/OpenSK/badge.svg?branch=develop)](https://coveralls.io/github/google/OpenSK?branch=develop)
 
+*News:*
+
+- 2023-08-24: [PQC paper reference](#Research)
+
 ## OpenSK
 
 This repository contains a Rust implementation of a
 [FIDO2](https://fidoalliance.org/fido2/) authenticator.
-We developed OpenSK as a [Tock OS](https://tockos.org) application.
+Also called security keys, they are a external devices that can be used for
+signing in on websites.
```

Comment:
This sentence is weird (starting with "also called security key" instead of a pronom).

Reply:
Replaced "authenticator" with "security key" and made the sentence easier and clearer.